### DISCUSSION TITLE
How is my intellectual property safe

### DISCUSSION THREAD

**Discussion by raidaddicts** [community/community#40888] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hey Folks, 

So i am just starting out on GitHub , and I have run this online game for about 20 years now and for that entire time I have never really allowed anyone to have full access neither READ nor WRITE.    I do have people that I trust and want to be able to see the code and assist me with it but I am curious is there anything that would protect me from them going rogue and just stealing all of my ideas and starting their own thing?

The only thing that initially came to my mind was maybe a NDA but perhaps there is something like this built into GITHUB that I could legally take action or do something to protect myself if this were to happen.

Again sorry for the n00b question but hopefully someone can answer me and let me know..

**Comment by radar-blip** [comment]
You could make a private repository. https://docs.github.com/en/repositories/creating-and-managing-repositories/about-repositories#about-repository-visibility

**Reply by raidaddicts** [reply]
Sure , private but what about those people that I may trust now but they end up taking my work down the road because I pissed them off or something?

**Reply by radar-blip** [reply]
&copy;
